components/patina_smbios/src/manager.rs

```diff
@@ -47,6 +47,7 @@ use self::protocol::{SmbiosProtocol, SmbiosProtocolInternal};
 ///
 /// Returns the protocol handle on success. The caller is responsible for managing the
 /// protocol lifetime (though in practice, UEFI protocols persist for the system lifetime).
+#[allow(dead_code)]
```

Comment:
Is this dead code? I assume you are always installing the protocol?

You've added a lot of allow dead code in this PR, but without removing calls to these functions. What changed?